### ISSUE TITLE
[llvm[clang-tools-extra]] Headers and libraries don't seem to be output

### ISSUE THREAD

**Issue by NickStrupat** [microsoft/vcpkg#18303]
****

**Is your feature request related to a problem? Please describe.**
When I install the LLVM port with the feature for `clang-tools-extra`, those headers and libraries do not show up in the installed directory.

**Proposed solution**
Fix the port package to build and output the appropriate headers and libraries for `clang-tools-extra`

**Describe alternatives you've considered**
I have tried including `clang` feature in my feature list, thinking maybe it might be a build dependency which isn't configured in the dependencies list of the features. This didn't work.

**Comment by PhoebeHui** [comment]
@NickStrupat, the tolls are located to installed/[triplet]/tools directory, could you double check?

**Comment by NickStrupat** [comment]
@PhoebeHui, I just checked and they're not there. My apologies, I should have specified that I'm using `vcpkg` in manifest mode. I don't see it in `vcpkg_installed/[triplet]/tools`. In fact, there is no `tools` sub-directory in there.

I also think I made a mistake in my understanding of these tools. I was hoping to use the language server implementation in `clangd` as a library. I didn't realize that the way `clangd` is structured, it is only built as an executable, not as a library.

**Comment by PhoebeHui** [comment]
@NickStrupat, yes clangd is built as executable not as a library. BTW I just tried, and it has tools folder.

vcpkg.json
```
{
    "name": "test",
    "version": "0.0.1",
    "dependencies": [
     {
      "name": "llvm",
      "features": [ "clang-tools-extra" ]
     }
    ]
}
```

**Comment by yurybura** [comment]
@NickStrupat After finish `.\vcpkg install llvm[clang-tools-extra]:x64-windows` you can find `.\installed\x64-windows\tools\llvm\clangd.exe`.
